data_steward/utils/participant_summary_requests.py

```diff
@@ -101,7 +120,64 @@ def get_deactivated_participants(project_id, columns):
                     item.append(val)
         deactivated_participants.append(item)
 
-    df = pandas.DataFrame(deactivated_participants,
+    # Transforms participantId to an integer string
+    updated_deactivated_participants = participant_id_to_int(deactivated_participants, columns)
+
+    df = pandas.DataFrame(updated_deactivated_participants,
                           columns=deactivated_participants_cols)
 
-    return df
+    # To store dataframe in a BQ dataset table
+    destination_table = dataset_id + '.' + tablename
+
+    dataset = store_participant_data(df, project_id, destination_table)
+
+    return dataset
+
+
+def participant_id_to_int(participant_list, columns):
+    """
+    Transforms the participantId received from RDR ParticipantSummary API from an
+    alphanumeric string to an integer string.
+
+    :param participant_list: list of participants fetched from ParticipantSummary API
+    :param columns: columns to be pushed to a table in BigQuery in the form of a list of strings
+    :return: returns the list of participants with the participantId transformed to
+                an integer string
+    """
+
+    # determines the index of participantId in columns list
+    participant_id = columns.index('participantId')
+
+    updated_participant_list = []
+
+    for participant in participant_list:
+        participant[participant_id] = int(participant[participant_id][1:])
+        updated_participant_list.append(participant)
+
+    return updated_participant_list
```

Comment:
I've been toying with pandas some more and think you could make this a pandas helper function that can just be applied to an existing column.  You can create custom functions to apply to columns like in this [stackoverflow post](https://stackoverflow.com/questions/50947447/convert-values-of-a-column-in-pandas).

To test, you can run this in an interpreter:
```
import pandas as pd

# create the dataframe
df = pd.DataFrame([['P111'],['P222'],['P333']], columns=['pid'])

# view dataframe
df

# view dataframe info
df.info(verbose=True)

# define custom function
def participant_id_to_int(pid):
    return int(pid[1:])

# apply the custom function
df['pid'] = df['pid'].apply(participant_id_to_int)

# view dataframe
df

# view dataframe info
df.info(verbose=True)
```
Then this could just be another transformation you apply to your dataframe without you needing to identify the column index of the `participant_id`.

Reply:
Oh, I like how concise this is!

Reply:
This definition is assuming the first character is always an alphabetic character.  You could add checks in.  It might not be necessary because if the format of this string changes, it will have to be communicated across multiple teams, as a lot of systems could break.  But for an example, you could use:

```
import string

def participant_id_to_int(pid):
    return int(pid.strip(string.ascii_letters))
```
This would perform a case-insensitive strip of all leading and trailing alphabetic characters.  You can find documentation in the [string API](https://docs.python.org/3.7/library/stdtypes.html?highlight=strip#string-methods) by searching for strip.